packages/app/src/files-manager/files-manager.container.tsx

```diff
@@ -9,6 +9,8 @@ import FilesManager from "App/files-manager/components/files-manager/files-manag
 
 const mapStateToProps = (state: RootState & ReduxRootState) => ({
   memorySpace: state.device.data?.memorySpace,
+  musicFiles: state.filesManager.files,
```

Comment:
```suggestion
  files: state.filesManager.files,
```

`MtpFileType` will be mark that file is music or not 😉